app/cli/cmd/workflow_contract_describe.go

```diff
@@ -100,3 +101,19 @@ func contractDescribeTableOutput(contractWithVersion *action.WorkflowContractWit
 
 	return nil
 }
+
+func stringifyAssociatedWorkflows(contractWithRevision *action.WorkflowContractWithVersionItem) string {
+	contract := contractWithRevision.Contract
+	if len(contract.WorkflowRefs) != 0 {
+		var workflows []string
+		for _, w := range contract.WorkflowRefs {
+			workflows = append(workflows, fmt.Sprintf("%s/%s", w.ProjectName, w.Name))
+		}
+		// sort the workflows
+		sort.Strings(workflows)
+
+		return strings.Join(workflows, "\n")
+	}
+
+	return strings.Join(contract.Workflows, ", ")
```

Comment:
I do not understand this method, is it maintaining both ways of showing it or smth?